app-network/mihomo-party/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=mihomo-party
+PKGSEC=net
+PKGDEP="gtk-3 mihomo nss libsecret libnotify xdg-utils at-spi2-core"
+BUILDDEP="nodejs"
+PKGDES="Another Mihomo GUI"
+
+PKGPROV="mihomo-party"
+
+
+# FIXME: Vite not supported on loongson3, riscv64,loongarch64.
+FAIL_ARCH="(loongson3|riscv64|loongarch64|ppc64el)"
```

Comment:
Perhaps take a page out of `clash-verge-rev`, which builds on LoongArch with a patch for rollup.

Reply:
Might as well go with:

```
FAIL_ARCH="!(amd64|arm64)"
```